stable/nginx-ingress/README.md

```diff
@@ -60,6 +60,7 @@ Parameter | Description | Default
 `controller.extraArgs` | Additional controller container arguments | `{}`
 `controller.kind` | install as Deployment or DaemonSet | `Deployment`
 `controller.tolerations` | node taints to tolerate (requires Kubernetes >=1.6) | `[]`
+`controller.minReadySeconds` | how many seconds a pod needs to be ready before killing the next, during update | `15`
```

Comment:
Could you please update the readme for the time changes as well?

Reply:
Done 👍